extmod/nimble/modbluetooth_nimble.c

```diff
@@ -808,6 +889,40 @@ int mp_bluetooth_set_preferred_mtu(uint16_t mtu) {
     return 0;
 }
 
+#if MICROPY_PY_BLUETOOTH_ENABLE_PAIRING_BONDING
+int mp_bluetooth_gap_pair(uint16_t conn_handle) {
+    DEBUG_printf("mp_bluetooth_gap_pair: conn_handle=%d\n", conn_handle);
+    return ble_hs_err_to_errno(ble_gap_security_initiate(conn_handle));
+}
+
+int mp_bluetooth_gap_passkey(uint16_t conn_handle, uint8_t action, mp_int_t passkey) {
+    struct ble_sm_io io = {0};
+
+    switch (action) {
+        case MP_BLUETOOTH_PASSKEY_ACTION_INPUT: {
+            io.passkey = passkey;
+            break;
+        }
+        case MP_BLUETOOTH_PASSKEY_ACTION_DISP: {
+            io.passkey = passkey;
+            break;
+        }
+        case MP_BLUETOOTH_PASSKEY_ACTION_NUMCMP: {
```

Comment:
`MP_BLUETOOTH_PASSKEY_ACTION_DISP` above and `MP_BLUETOOTH_PASSKEY_ACTION_NUMCMP` don't match the defines